NavigationReactNative/src/ios/NVSceneController.m

```diff
@@ -47,14 +46,23 @@ - (void)viewWillAppear:(BOOL)animated
         [self.navigationItem setTitle:navigationBar.title];
     }
 
-    [navigationBar updateColors];
+    [navigationBar updateTintColors];
+    [navigationBar updateTitleColors];
 
     if (@available(iOS 11.0, *)) {
         self.navigationController.navigationBar.prefersLargeTitles = true;
         [self.navigationItem setLargeTitleDisplayMode:navigationBar.largeTitle ? UINavigationItemLargeTitleDisplayModeAlways : UINavigationItemLargeTitleDisplayModeNever];
     }
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:animated];
+    
+    // This is seemingly required, otherwise the small navbar title color does not get reset when only called from viewWillAppear
+    NVNavigationBarView *navigationBar = (NVNavigationBarView *) [self.view viewWithTag:NAVIGATION_BAR];
+    [navigationBar updateTitleColors];
```

Comment:
Do we need a separate `updateTitleColors`? Why not call `updateColors` again? I know that we don't need to update the tint colors but it doesn't cost anything

Reply:
Yeah it doesn't hurt anything, was just trying to cut down on duplicate calls